Makefile

```diff
@@ -1,12 +1,23 @@
 # Compiler flags
 CFLAGS := -Wall -Wextra -O3 -m64 -D'uint64_t=unsigned long long'
 
+# Give Selfie information about the host operating system
+ifeq ($(OS),Windows_NT)
+    CFLAGS += -D'HOST_OS=3ull'
+else
```

Comment:
Can this be done in a single line simply passing $OS? $OS on my system is "OSX". Even if this doesn't work I would still prefer if we could do this just based on $OS and not additional stuff like `uname`.

Reply:
Good point, I'll try that.